core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol3/deprecationCause.jelly

```diff
@@ -0,0 +1,6 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+  ${%This protocol is unstable. See the protocol documentation for more info.}
+  <!-- TODO: Move/mirror it to jenkins.io -->
+  <a href="https://github.com/jenkinsci/remoting/blob/master/docs/protocols.md#jnlp3-connect-errata">${%JNLP3 Protocol Errata}</a>
```

Comment:
OK, I have the requested links in https://github.com/jenkinsci/jenkins/pull/2950. Now we just need to agree where to keep them, CC @rtyler